pkg/apis/serving/v1alpha2/kfservice_types.go

```diff
@@ -149,6 +149,7 @@ type StatusConfigurationSpec struct {
 
 // KFService is the Schema for the services API
 // +k8s:openapi-gen=true
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
```

Comment:
checked knative they also print the reason field, what do you think adding that as well?
```
+kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].reason"
```

Reply:
@yuzisun Personally I think we can just show overall status. some reasons: 1. User executes `kubectl get kfservice` just want to get the status (if ready or not), if add the reasons, may be too long for user quick checking. 2. If I were user, I would `kubectl describe` or `log` to check the details, and if have reasons, I guess that may be long.  3. Checked other CRD such as `tfjobs`, just shows status without reasons. :-) 
Of course, if you persist to that, I'm happy to add that. Thanks.

Reply:
Is reason available on kfservice describe? I have found that I have to look at underlying configuration or revision or pod to figure out reason... not sure if this is related to your current change though?

Reply:
I agree with @jinchihe, it's already pretty long. I think the primary information is conveyed well, more details can always be retrieved without short output.

Reply:
> Is reason available on kfservice describe? I have found that I have to look at underlying configuration or revision or pod to figure out reason... not sure if this is related to your current change though?

The reason should be already available on KFService status conditions, I am not very sure it shows up on `kubectl describe`.